cmd/rest.go

```diff
@@ -19,13 +20,15 @@ func startRestServerCmd() *cobra.Command {
 			cfg, err := config.ParseConfig()
 			common.PanicIfError(err)
 			r, err := rest.LoadRouters(cfg)
+			originsOk := handlers.AllowedOrigins([]string{"*"})
```

Comment:
Why we are doing CORS in the coordinator client? What's the use case here? Usually, this should be handled in a gateway nginx or something in the server that the coordinator client is deployed.

Reply:
Yeah the next step would be allowing to configure these properties in the `config.toml` so a user has options to run commander by itself or behind an API gateway for eg: nginx

Reply:
It just seems right to allow users to configure the HTTP server itself with the additional option of putting it behind a API gateway and configuring that.

Open to changing this.

Reply:
Per our offline discussion, let's add cors to unblock wallet development and fix it later. #120 is the issue for this.